src/AM.Condo/Targets/Test/DockerCompose.targets

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xlmns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<Target Name="DockerComposeTest" Condition=" $(DockerRequired) AND $(DOCKER_COMPOSE_PATHS) != '' ">
+        <!-- .sh file needs to be created to be run directly by bash with #!/bin/bash instead of dash used by agent -->
+        <!-- Init bash script in debug mode -->
+		<Exec Command="echo &apos;#!/bin/bash -x&apos; > compose.sh" />
```

Comment:
What and where is this compose.sh file?
why are we running in debug mode?

Reply:
compose.sh is temporarily being created in the current working directory and later deleted (line 18)

Reply:
We are running in debug mode so that the .sh script outputs the performed commands

Reply:
simplified call, there's no need to run bash scripts